src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -120,6 +120,18 @@
         <type>Number</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>RecurringDonationNaming__c</fullName>
+        <description>The format to use for Recurring Donation naming.</description>
```

Comment:
@sam-knox Can you review the description and inline help text for this RD Setting custom field. It is used to store the RD auto naming format.